develop/plone/searching_and_indexing/external-catalogs.rst

```diff
@@ -0,0 +1,58 @@
+=======================
+Using external catalogs
+=======================
+
+.. admonition:: Description
+
+        The Plone catalog can be extend to use external catalogs like Solr or elastic search. Add-ons like collective.solr use that to hook into the catalog API and do some Indexing outside of Plone in Solr, which increases performance and flexibility of indexing a lot.
```

Comment:
According to their own website it should be _Elasticsearch_ not _elastic search_